src/bicep/examples/appServicePlan/appService.bicep

```diff
@@ -0,0 +1,50 @@
+/*
+Deployes a web server farm(aka: App Service Plan) to support web container deployments for Linux.  
+Optionally enable dynamic auto scaling based on CPU Percentages using 'enableAutoScale' true/false
+*/
+targetScope = 'subscription'
+
+param mlzDeploymentVariables object = json(loadTextContent('../deploymentVariables.json'))
+
+@description('The name of the web server farm which will be created.  If unchanged or not specified, the MLZ resource prefix + "--ASP" will be utilized.')
+param appServicePlanName string = '${mlzDeploymentVariables.mlzResourcePrefix.Value}-asp'
+
+@description('The name of the resource group in which the app service plan will be deployed. If unchanged or not specified, the MLZ shared services resource group is used.')
+param targetResourceGroup string = '${mlzDeploymentVariables.spokes.Value[2].resourceGroupName}'
+
+@description('If true, enables dynamic scale-in & scale-out based on CPU percentages.  If false, then compute instances remain static with 2 instances supporting all traffic')
+param enableAutoScale bool = true
+
+var targetSubscriptionId_Var = contains(targetResourceGroup, '${mlzDeploymentVariables.spokes.Value[2].resourceGroupName}') ? '${mlzDeploymentVariables.spokes.Value[2].subscriptionId}' : subscription().subscriptionId
```

Comment:
I think we want an equality comparison here, right? Not just that the parameter contains the same string?

```
var targetSubscriptionId_Var = targetResourceGroup == mlzDeploymentVariables.spokes.Value[2].resourceGroupName ? mlzDeploymentVariables.spokes.Value[2].subscriptionId : subscription().subscriptionId
```

The likelihood is low that it could happen, but an `rg-fizz` is contained in `rg-fizz-buzz`

Reply:
Yes sir, completed.